core/server/common/src/test/java/alluxio/master/journal/raft/SnapshotReplicationManagerTest.java

```diff
@@ -120,7 +120,7 @@ public void before() throws Exception {
 
   private SimpleStateMachineStorage getSimpleStateMachineStorage() throws IOException {
     RaftStorage rs = new RaftStorageImpl(mFolder.newFolder(CommonUtils.randomAlphaNumString(6)),
-            RaftServerConfigKeys.Log.CorruptionPolicy.getDefault());
+            RaftServerConfigKeys.Log.CorruptionPolicy.getDefault(), 0);
```

Comment:
Are these changes have related with this PR?

Reply:
Aligned ratis version, these changes are needed.